src/handleAnswers.js

```diff
@@ -35,14 +35,16 @@ const createPublicProp = (x) => (x.publicPathRequired && {
 })
 
 // loadersMap :: Object -> String -> Object
-const loadersMap = (config) => (x) => (config[x.trim()])
+const loadersMap = (config) => (x = '') => (config[x.trim()])
 
 // createLoadersConfig :: Object -> Object
 // extend answers object with mapped loader configs
-const createLoadersConfig = (x) => ({
-  ...x,
+const createLoadersConfig = ({initialEntry ="", loaders = [], ...rest}) => ({
```

Comment:
Nit: I prefer to use single quotes for consistency and it looks like you used default quotes above. Can you update this to be single quotes?